stdlib/public/core/KeyPath.swift

```diff
@@ -166,6 +243,33 @@ public class AnyKeyPath: Hashable, _AppendKeyPath {
     return try f(KeyPathBuffer(base: base))
   }
 
+  // TODO: Find a quicker way to see if this is a tuple.
+  internal func isTuple(_ item: Any) -> Bool {
```

Comment:
Can this function be removed now?

Reply:
I'm still using it in `_swift_getKeyPath` as of the latest commit, but I'll see if my current processing inside `visitStoredComponent` is filtering these out correctly.

Reply:
Interesting, looks like the tests pass without it! I'll have it removed in the next commit.

Reply:
The latest commit resolves this.